src/askem_beaker/contexts/mira_model_edit/agent.py

```diff
@@ -525,6 +525,8 @@ async def stratify(self,
                 An iterable of pairs corresponding to a directed network structure
                 where each of the pairs has two strata. If none given, will assume a complete
                 network structure. If no structure is necessary, pass an empty list.
+                For example [["Young", "Old"]] would mean that the population in Young can interact with the population in Old provided they are within the same state.
```

Comment:
I think this could clearer:
> This describes how different groups within the same strata of a state can convert into each other.
> It is an iterable of pairs corresponding to the edge list of a directed network where the nodes are the different stratum groups.
> If the network is complete, where every stratum group can convert into any other, then give `None`
> If no conversion between any stratum group is allowed, then give an empty list `[]`. An example of no conversion is stratification with immutable demographic traits like age group: individuals do not change age groups during the simulation and therefore `structure = []`.
> A non-trivial `structure` could be a travel network where travel is possible between physically adjacent countries; for example, `structure = [["Portugal", "Spain"], ["Spain", "France"], ["France", "Germany"]]`, where the pairs `["Portugal", "France"]` etc. are excluded because those pairs of countries are not adjacent.

The point I want to make is that `structure` is specifically about 'conversion', not just 'interaction'.